absl/synchronization/internal/futex.h

```diff
@@ -79,6 +79,12 @@ namespace synchronization_internal {
 
 #if defined(SYS_futex_time64) && !defined(SYS_futex)
 #define SYS_futex SYS_futex_time64
+using futex_timespec = struct timespec;
+#else
+struct futex_timespec {
```

Comment:
Is there a reason not to make this unconditional?

Reply:
I think I understand why this is done now, but code comments would be helpful for future readers.